uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.service.spec.js

```diff
@@ -1,82 +1,86 @@
 /* eslint-env amd, jasmine */
+/* global Drupal */
 
-define([
-  'common/angular',
-  'common/lodash',
-  'common/angularMocks',
-  'leave-absences/calendar-feeds/link-modal/calendar-feeds.link-modal.module'
-], function (angular, _) {
-  'use strict';
+(function (Drupal) {
+  define([
+    'common/angular',
+    'common/lodash',
+    'common/angularMocks',
+    'leave-absences/calendar-feeds/link-modal/calendar-feeds.link-modal.module'
+  ], function (angular, _) {
+    'use strict';
 
-  describe('CalendarFeedsLinkModalService', function () {
-    var $rootScope, $uibModal, CalendarFeedsLinkModal,
-      calendarFeedsLinkModalComponent;
+    describe('CalendarFeedsLinkModalService', function () {
+      var $rootScope, $uibModal, CalendarFeedsLinkModal,
+        calendarFeedsLinkModalComponent;
 
-    beforeEach(angular.mock.module('calendar-feeds.link-modal', function ($compileProvider,
-      $provide) {
-      mockCalendarFeedsLinkModalComponent($compileProvider, $provide);
-    }));
+      beforeEach(angular.mock.module('calendar-feeds.link-modal', function ($compileProvider,
+        $provide) {
+        mockCalendarFeedsLinkModalComponent($compileProvider, $provide);
+      }));
 
-    beforeEach(inject(function (_$rootScope_, _$uibModal_, _CalendarFeedsLinkModal_) {
-      $rootScope = _$rootScope_;
-      $uibModal = _$uibModal_;
-      CalendarFeedsLinkModal = _CalendarFeedsLinkModal_;
+      beforeEach(inject(function (_$rootScope_, _$uibModal_, _CalendarFeedsLinkModal_) {
+        $rootScope = _$rootScope_;
+        $uibModal = _$uibModal_;
+        CalendarFeedsLinkModal = _CalendarFeedsLinkModal_;
 
-      spyOn($uibModal, 'open').and.callThrough();
-    }));
+        spyOn($uibModal, 'open').and.callThrough();
+      }));
 
-    it('is defined', function () {
-      expect(CalendarFeedsLinkModal).toBeDefined();
-    });
+      it('is defined', function () {
+        expect(CalendarFeedsLinkModal).toBeDefined();
+      });
 
-    describe('open()', function () {
-      var url = 'https://civihr.org/';
+      describe('open()', function () {
+        var hash = 'jahmaljahsurjahber';
+        var expectedFeedUrl = Drupal.absoluteUrl('/') + 'civicrm/calendar-feed?hash=' + hash;
```

Comment:
Inject HOST_URL (or BASE_URL) instead of using Drupal directly. If in the future the way the constant is defined changes then we won't need to change this.